drivers/sensor/bosch/bmp388/bmp388.h

```diff
@@ -19,16 +19,33 @@
 #include <zephyr/drivers/sensor.h>
 #include <zephyr/sys/util.h>
 
-#define DT_DRV_COMPAT bosch_bmp388
+#ifdef DT_DRV_COMPAT
+#undef DT_DRV_COMPAT
+#endif
 
+#define DT_DRV_COMPAT bosch_bmp388
 #define BMP388_BUS_SPI DT_ANY_INST_ON_BUS_STATUS_OKAY(spi)
 #define BMP388_BUS_I2C DT_ANY_INST_ON_BUS_STATUS_OKAY(i2c)
+#undef DT_DRV_COMPAT
+
+#define DT_DRV_COMPAT bosch_bmp390
+#define BMP390_BUS_SPI DT_ANY_INST_ON_BUS_STATUS_OKAY(spi)
+#define BMP390_BUS_I2C DT_ANY_INST_ON_BUS_STATUS_OKAY(i2c)
+#undef DT_DRV_COMPAT
+
+#if defined(BMP388_BUS_SPI) || defined(BMP390_BUS_SPI)
+#define USE_SPI_BUS
+#endif
+
+#if defined(BMP388_BUS_I2C) || defined(BMP390_BUS_I2C)
+#define USE_I2C_BUS
+#endif
 
 union bmp388_bus {
-#if BMP388_BUS_SPI
+#if defined(USE_SPI_BUS)
```

Comment:
`#ifdef`

Reply:
make that for all bmp388*.c files  where I make use of USE_XXX_BUS